README.md

```diff
@@ -268,7 +268,11 @@ This object defines templates (functions) that are used for displaying parts of
 
 `inputValue` is a function that receives one argument, the currently selected suggestion. It returns the string value to be inserted into the input.
 
-`suggestion` is a function that receives one argument, a suggestion to be displayed. It is used when rendering suggestions, and should return a string, which can contain HTML. :warning: **Caution:** because this function allows you to output arbitrary HTML, you should [make sure it's trusted](https://en.wikipedia.org/wiki/Cross-site_scripting), and accessible.
+`suggestion` is a function that receives one argument, a suggestion to be displayed. It is used when rendering suggestions, and should return a string, which can contain HTML.
```

Comment:
Sorry in advance if I've misunderstood - I'm guessing the suggestion is displayed when the user types into a text-field? (Let me know if this is one of those cases where using the passive voice is preferable to writing something contorted.)

Maybe something like this could work:

> `suggestion` is a function that receives one argument, a suggestion for the dropdown to display. The autocomplete uses this function when rendering suggestions. `suggestion` should return a string, which can contain HTML.

Reply:
I haven't added this line in this PR – just split it up. However, whilst we're here, I agree it could be improved and your suggestion's a really good start.

I think it'd probably make sense to swap the order around – at the minute we talk about what the function takes as input, what it's for, and what it should output… which seems muddled.

What about something like…

> `suggestion` can be used to customise how suggestions are rendered in the dropdown. It must be a function which will be called with one argument, which will be the suggestion. The function must return a string, which can contain HTML.

Reply:
The more I pick at this the more I want to rewrite the whole section on `templates` – possibly trying to cover off #420 at the same time.

I feel like that's starting to really stretch the scope of this PR, so how would you feel about me adding the new line in without making changes to the existing content, and creating a new issue or PR to tackle the surrounding docs?

Reply:
@EoinShaughnessy any objections to leaving further improvements to the existing docs out for now?

Reply:
@36degrees No objections! Sorry for late reply. Give me a shout whenever we want to tackle the follow-up issue.